wallet/test/widget_testing/pages/accept_policy/accept_policy_screen_test.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:get_it/get_it.dart';
+import 'package:mockito/mockito.dart';
+import 'package:pylons_wallet/pages/presenting_onboard_page/screens/accept_policy_screen.dart';
+import 'package:pylons_wallet/pages/presenting_onboard_page/viewmodel/accept_policy_viewmodel.dart';
+import 'package:pylons_wallet/pages/purchase_item/purchase_item_screen.dart';
+import 'package:pylons_wallet/pages/purchase_item/purchase_item_view_model.dart';
+import 'package:pylons_wallet/utils/constants.dart';
+import '../../../mocks/accept_policy_viewmodel.mocks.dart';
+import '../../../mocks/mock_constants.dart';
+import '../../../mocks/purchase_item_viewmodel.mocks.dart';
+import '../../extension/size_extension.dart';
+
+void main() {
+  final acceptPolicyViewModel = MockAcceptPolicyViewModel();
+  final purchaseItemViewModel = MockPurchaseItemViewModel();
+  GetIt.I.registerLazySingleton<AcceptPolicyViewModel>(() => acceptPolicyViewModel);
+  GetIt.I.registerLazySingleton<PurchaseItemViewModel>(() => purchaseItemViewModel);
+
+  testWidgets(
+    "is accept policy portion showing",
+    (tester) async {
+      await tester.testAppForWidgetTesting(
+        AcceptPolicyScreen(
+          nft: MOCK_NFT_FREE_IMAGE,
+        ),
+      );
+      await tester.pump();
+      final kAcceptPolicyPortion = find.byKey(const Key(kAcceptPolicyPortionKey));
+      expect(kAcceptPolicyPortion, findsOneWidget);
+    },
+  );
+
+  testWidgets(
+    "can user tap on get started button to accept policies",
```

Comment:
The test can be better. 
`When the terms and conditions are not accepted then user cannot go to next screen.`
`When the terms and conditions are accepted then user can go to next screen.`

Reply:
sure, will add these two